stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -3437,10 +3450,10 @@ void YangParseTreePaths::AddSubtreeInterfaceFromSingleton(
     const SingletonPort& singleton, const NodeConfigParams& node_config,
     YangParseTree* tree) {
   const std::string& name =
-      !singleton.name().empty()
-          ? singleton.name()
-          : absl::StrFormat("%d/%d/%d", singleton.slot(), singleton.port(),
-                            singleton.channel());
+      singleton.name().empty()
+          ? absl::StrFormat("%d/%d/%d", singleton.slot(), singleton.port(),
```

Comment:
Should we also add a prefix to the singleton port? since we have the prefix for the optical port, node, and chassis.

Reply:
Good question. This scheme comes from the name we currently have in all shipped ChassisConfigs, so I'd assume that they do not cause issues.

Reply:
ok, that's fine to me